spark/streamingGuide1/start-streamingGuide1-LinearRegKafka.sh

```diff
@@ -0,0 +1,11 @@
+echo '...Building Spark Streaming App '
+
+sbt compile
+
+sbt "run-main LinearRegKafka" 2>&1 1>$STREAMAI_HOME/logs/spark/streamingguide1/linearregkafka.log &
+
+sleep 1m
```

Comment:
without putting the sleep, I was hitting a weird behaviour whereby the  previous statement ('run-main') was not getting executed.  I presume the problem could be because I am asking the job to run as a background process (note the '&' at the end), so the script was terminating before the background process was started. 

putting a sleep before terminating the script solved the issue.